.travis/install.sh

```diff
@@ -50,3 +50,15 @@ pip install -r requirements.d/development.txt
 pip install codecov
 python setup.py --version
 pip install -e .[fuse]
+
+if [ -d "$HOME/.rustup/toolchains" ] && [ -f "$HOME/.cargo/env" ]; then
+    source ~/.cargo/env
+    rustup toolchain install stable # also updates
+    rustup default stable
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source ~/.cargo/env
+fi
+
+cd test-wrapper
+cargo build --release
```

Comment:
yes, my usage would be in vagrant, outside of travis.

maybe these lines could live in scripts/build-testwrapper or so and could be just called from install.sh?